packages/embed/src/lib/nfid.spec.ts

```diff
@@ -2,6 +2,17 @@ import { NFID } from './nfid';
 import { NfidAuthClient } from './authentication/auth-client';
 import { buildIframe } from './iframe/make-iframe';
 
+const TIMEOUT = 30000;
```

Comment:
If we do it this way, we should immediately dispatch the event. This is just increasing the pipeline runs by 30 sec.

Reply:
Is it possible to even dispatch it immediately?